ui/imports/shared/controls/chat/ProfileHeader.qml

```diff
@@ -115,11 +116,14 @@ Item {
         }
 
         RowLayout {
-            spacing: Style.current.halfPadding
+            spacing: compact ? 4 : Style.current.halfPadding
+            Layout.fillWidth: true
             Layout.alignment: Qt.AlignHCenter
             visible: root.displayNamePlusIconsVisible
             StyledText {
+                Layout.maximumWidth: root.width - Style.current.xlPadding
```

Comment:
There is several layouts in object tree between this text item and root. 
```
Layout.fillWidth: true
Layout.rightMargin: Style.current.xlPadding
```
Should look the same, and there is no need to set `Layout.fillWidth: true` for RowLayout

Reply:
You need to set it in order to get the `elide` working, otherwise with long user names, it would overflow; this is how it looked without the fix:

![Snimek_obrazovky_z_2022-07-21_10-37-49](https://user-images.githubusercontent.com/5377645/180225797-06c2f6d2-75d0-4fa3-92b7-6ad31dab662b.png)

Reply:
> and there is no need to set Layout.fillWidth: true for RowLayout

TIL, could you please point to the documentation? I can't find it :thinking:

Reply:
https://doc.qt.io/qt-5/qml-qtquick-text.html#elide-prop

> The text will only elide if an explicit width has been set.

Reply:
That means any of `width`, `anchors.left+anchors.right` or `Layout.prefered/fillWidth`